apis/dataprotection/v1alpha1/backuppolicy_types.go

```diff
@@ -105,6 +105,22 @@ type BackupTarget struct {
 	//
 	// +kubebuilder:validation:Required
 	ServiceAccountName string `json:"serviceAccountName,omitempty"`
+
+	// Specifies the container port in the target pod.
+	//
+	// +optional
+	ContainerPort *ContainerPort `json:"containerPort,omitempty"`
```

Comment:
as above

Reply:
It seems that PortName and ConnectionCredential.PortKey 're representing the same info.

Reply:
There is no need to add an addtional ContainerPort field.

Reply:
ConnectionCredential.PortKey specifies the map key of the port in the connection credential secret, and since kb0.9, the connection credential secret is changed to system account secret, which doesn't contain port key and value.

Reply:
I see,  ConnectionCredential.PortKey is set by buildBackupTarget, working with legacy `clusterDefinition.spec.ConnectionCredential` API.